content/ew-issue-z200-[2017-03-29].yaml

```diff
@@ -101,30 +125,68 @@ content:
                 description: ""
                 link: https://www.youtube.com/watch?v=ZCHFjGSCqP4
             -
+                title: "Ember.js' future, with Tom Dale"
+                description: ""
+                link: https://www.linkedin.com/pulse/emberjs-future-tom-dale-greg-leffler
+            -
                 title: "Creating an Ember Addon - Scott Batson"
                 description: ""
                 link: https://www.youtube.com/watch?v=bhJ6YzBIoWo
+            -
+                title: "Audacious Enterprise GIS Apps with Ember.js"
+                description: ""
+                link: https://www.youtube.com/watch?v=7Bvr7cEQNCg
+    
+    -
+        heading: "So Ember 2017 Videos"
```

Comment:
So Ember 2017 Conference videos just got posted. If there's enough related content and/or a section is getting quite big, I typically split it out into it's own section like this.